app/3.0/docs/tools/create-app-polymer-cli.md

```diff
@@ -74,29 +83,33 @@ application-specific elements should be defined in the `src` directory, at the s
 <pre><code>app/
   src/
     <var>my-app</var>/
-      <var>my-app</var>.html
+      <var>my-app</var>.js
     <var>my-el</var>/
-      <var>my-el</var>.html</code></pre>
+      <var>my-el</var>.js</code></pre>
 
 To add another element to the project:
 
 1. Create a new folder under `src`.
    
     <pre><code>mkdir src/<var>my-el</var></code></pre>
    
-2. Create an HTML import for the new element. You can use the existing app element as a starting point.
+2. Create a JavaScript module for the new element. You can use the existing app element as a starting point.
    
-3. To use the new element, you'll need to import it into your application element (for example, <code><var>my-app</var>.html</code>) with an "import" link:
+3. To use the new element, you'll need to import it into your application element (for example, <code><var>my-app</var>.js</code>) with an "import" statement:
    
-    <pre><code>&lt;link rel="import" href="/src/<var>my-el</var>/<var>my-el</var>.html"&gt;</code></pre>
+    <pre><code>import '<var>my-el</var>/<var>my-el</var>.js';</code></pre>
    
    Don't use `polymer init` to create an element project inside your app project.
 
 ## Manage dependencies in an application project {#dependencies}
 
-Applications should use relative URLs to import other source files and dependencies. But because 
-applications are served independently, they can properly reach into the dependencies directory for 
-dependencies.
+Import your dependencies with module specifiers:
+
+src/my-app.js {.caption}
+
+```
+import '@polymer/polymer/polymer-element.js';
+import 'my-el/my-el.js';
```

Comment:
Import for a local resource must start with ./ ../ or / ... So this should be `../my-el/my-el.js` or `/src/my-el/my-el.js`